components/Form/Form.unit.test.jsx

```diff
@@ -179,6 +179,54 @@ describe('Form component ', () => {
 
       expect(onValidSubmitCallback).toHaveBeenCalled();
     });
+
+    it('Should exec validation in nested input ', () => {
+      const errorMsg = 'Valor mínimo de caracteres não alcançado';
+      const formNested = (
+        <Form onValidSubmit={onValidSubmitCallback} onSubmit={onSubmitCallback}>
+          <div>
+            <div>
+              <Input
+                name="name"
+                label="Name"
+                minLength="20"
+                validate={[
+                  validations.Required,
+                  {
+                    validate: validations.MinLength,
+                    error: errorMsg,
+                  },
+                ]}
+              />
+            </div>
+          </div>
```

Comment:
@pedrosouza what do you think to add a `<Input />` inside the parent `<div>`? And add another component to form (example Dropdown or RadioGroup).